samples/snippets/fsharp/azure/blob-storage.fsx

```diff
@@ -172,32 +135,34 @@ ListBlobsSegmentedInFlatListing container |> Async.RunSynchronously
 //
 
 // Get a reference to a container.
-let appendContainer = blobClient.GetContainerReference("my-append-blobs")
+let appendContainer = service.GetBlobContainerClient("my-append-blobs")
 
 // Create the container if it does not already exist.
 appendContainer.CreateIfNotExists() |> ignore
 
 // Get a reference to an append blob.
-let appendBlob = appendContainer.GetAppendBlobReference("append-blob.log")
+let appendBlob = appendContainer.GetAppendBlobClient("append-blob.log")
 
 // Create the append blob. Note that if the blob already exists, the 
 // CreateOrReplace() method will overwrite it. You can check whether the 
 // blob exists to avoid overwriting it by using CloudAppendBlob.Exists().
-appendBlob.CreateOrReplace()
+appendBlob.CreateIfNotExists()
 
 let numBlocks = 10
 
 // Generate an array of random bytes.
 let rnd = new Random()
-let bytes = Array.zeroCreate<byte>(numBlocks)
-rnd.NextBytes(bytes)
+let bytesArray = Array.zeroCreate<byte>(numBlocks)
+rnd.NextBytes(bytesArray)
 
 // Simulate a logging operation by writing text data and byte data to the 
 // end of the append blob.
 for i in 0 .. numBlocks - 1 do
-    let msg = sprintf "Timestamp: %u \tLog Entry: %d\n" DateTime.UtcNow bytes.[i]
-    appendBlob.AppendText(msg)
+    let msg = sprintf "Timestamp: %A \tLog Entry: %d\n" DateTime.UtcNow bytesArray.[i]
```

Comment:
We should consider updating this to use string interpolation.  _([ref](https://docs.microsoft.com/dotnet/fsharp/language-reference/interpolated-strings))_